### PR TITLE
[wip] Add pt-br translation for 'Session Secret Security'

### PR DESCRIPTION
**Pull request by cmatiello**
Add pt-br translation for 'Session Secret Security' item.

### CODE REVIEW

File: README.pt-br.md [hunk 1]
```diff
@@ -1295,6 +1296,31 @@ get '/:value' do
   session['value'] = params['value']
 end
 ```
+#### Chave de segurança da sessão
+
+Para melhorar a segurança, os dados da sessão no cookie são assinados com a chave de segurança da sessão
+usando `HMAC-SHA1`. Essa chave deve ser preferencialmente um
+valor randômico, criptograficamente seguro, com tamanho apropriado para
+`HMAC-SHA1`, maior ou igual a 64 bytes (512 bits, 128
+caracteres hexadecimais). Você seria orientado a não usar uma chave randômica menor que 32
```
Comment (talyssonoc) [inline README.pt-br.md]: I would change it to be "Aconselha-se que não se use [...]"

Comment (cmatiello) [reply README.pt-br.md]: I agree!

**Comment by namusyaka** [comment]
Any progress? Looks like still work in progress right?

**Comment by jkowens** [comment]
@cmatiello, @talyssonoc, @wenderjean would one of you be able to resolve the conflict? I'll get this merged once that is done, thanks!

**Comment by wenderjean** [comment]
I'll wait for a day and if @cmatiello cannot rebase it I'll try to do it or open another PR.

**Pull request closed without merging**